### PR TITLE
[documentation] #2193: Update Parity Scale Decoder Tool documentation

### PR DESCRIPTION
**Pull request by outoftardis**
Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

### Description of the Change

Updated README for Parity Scale Decoder Tool: https://github.com/outoftardis/iroha/tree/doc-psd/tools/parity_scale_decoder#parity-scale-decoder-tool

### Issue

Partially addresses #2193

### CODE REVIEW

File: tools/parity_scale_decoder/src/generate_map.rs [hunk 1]
```diff
@@ -26,7 +26,7 @@ impl<T> TypeName for T {
     }
 }
 
-/// Newtype which has `type_name()` method when `T` implements [`IntoSchema`]
+/// New type which has `type_name()` method when `T` implements [`IntoSchema`]
```
Comment (appetrosyan) [inline tools/parity_scale_decoder/src/generate_map.rs:29]: This is a technical term. Please leave as is.

Comment (outoftardis) [reply tools/parity_scale_decoder/src/generate_map.rs:29]: ok!

**Review by appetrosyan — DISMISSED** [review]

File: tools/parity_scale_decoder/README.md [hunk 2]
```diff
@@ -1,72 +1,86 @@
...
-
-Building:
+To build the tool, run:
 
 ```bash
 cargo build --bin parity_scale_decoder
 ```
 
-If your terminal does not support colors:
+If your terminal does not support colours, run:
```
Comment (appetrosyan) [inline tools/parity_scale_decoder/README.md:13]: Very good!

**Review by appetrosyan — APPROVED** [review]

**Review by Arjentix — APPROVED** [review]
Much better than it was before

**Pull request merged**